hat/geo.py

```diff
@@ -347,10 +347,14 @@ def get_latlon_keys(ds):
     return lat_key, lon_key
 
 
-def latlon_coords(ds):
+def latlon_coords(ds, names: list = []):
```

Comment:
Maybe dims instead?

Reply:
actually, this was correct. I mixed it up the netcdf dim names and station coordinates. I can revert it, or just support another optional input about the dim names. What do you think?

Reply:
Ah, I would revert it then!